docs/hub/spaces-advanced-usage.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Spaces Advanced Usage
+---
+
+<h1>Spaces Advanced Usage</h1>
+
+This section highlights some additional miscellaneous features that you can use to get the most out of Hugging Face Spaces.
+
+## Using OpenCV in Spaces
+
+In order to use OpenCV, you'll need to add it to both the `packages.txt` and `requirements.txt` files in your repository. If those files don't exist, you'll need to create them. For example, [see this sample repository](https://huggingface.co/spaces/templates/gradio_opencv/tree/main).
```

Comment:
Would this be better as a sub-section in the Custom Python Spaces section in `spaces-sdks.md`? Seems like you also have to select the Gradio SDK to run OpenCV.

Reply:
Ah nope, you don't need to use Gradio. OpenCV will work with Streamlit as well (or the custom Python backends) as far as I know, the linked example just happens to use Gradio. I'll clarify that!